src/Sulu/Bundle/LocationBundle/Resources/js/containers/Location/LocationOverlay.js

```diff
@@ -218,7 +218,7 @@ class LocationOverlay extends React.Component<Props> {
                         </Form.Field>
 
                         <Form.Field>
-                            <Map
+                            <MapContainer
```

Comment:
i think we need to at least adjust the following things:
- use `eventHandlers` prop instead of `on*` props for all `react-leaflet` components
- use [MapConsumer](https://react-leaflet.js.org/docs/api-map#mapconsumer) component to gather underlying `Map` instance to change center/zoom of map instead of using `center` and `zoom` prop of `MapContainer`